### ISSUE TITLE
Style suddenly revert to default

### ISSUE THREAD

**Issue by SyazAmirin** [pojome/elementor#6733]
Like the title said. While making the column and the text and all that, the style is what it should be but, when I update or preview, (the eyes icon) the heading turns back into colour blue which is the default colour.

You can see yourself here: http://hebatnya.com/justeru-impian-di-jaring

Note: See the below one. The last one. The last heading.

Actually, I've had this issue over a year ago but the workaround is too delete the column and create from scratch. I don't want to do that. It's a lot to do.

**Comment by SyazAmirin** [comment]
What?? That IS a bug! It reverts without me knowing! Not me who changes it!

**Comment by jarededdy** [comment]
I agree, that is a bug. I don't know why @shilo-ey disagreed and closed this. I'm also having phantom revision reverting. As in I'll update pages. They'll be live and all is well. Then some how those changes get reverted to old versions. I thought maybe one of my team members was doing this unintentionally but it happened one time when I was working on the page! Also I can confirm, I was the only one working on the page. Something is buggy about revisions. It doesn't happen often but it does happen and the revision history shows nothing of the changes I made. Almost as if the entire database was restored to a previous version.
